src/hooks/useProfileImageUpload.ts

```
@@ -0,0 +1,43 @@
+
+  const resetImage = () => {
+    setProfileImage(null);
+  };
+
+  return {
+    profileImage,
+    isLoading,
+    uploadImage,
+    resetImage,
+  };
+  return { profileImage, isLoading, uploadImage, resetImage } as const;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)